mono/metadata/object-forward.h

```diff
@@ -8,6 +8,35 @@
 #ifndef __MONO_OBJECT_FORWARD_H__
 #define __MONO_OBJECT_FORWARD_H__
 
+#include <mono/utils/mono-publib.h>
+
 typedef struct _MonoReflectionTypeBuilder MonoReflectionTypeBuilder;
 
+struct _MonoException;
```

Comment:
This is not needed.

Reply:
I once saw this matter and I've been a bit nervous since. It had to do with C++ namespaces and was years ago and nobody seems concerned and I haven't had the problem since. You see both styles out in the world -- forward declare and then typedef, and combined.